### ISSUE TITLE
uploading ifc files is not supported

### ISSUE THREAD

**Issue by kanbang** [pattern-x/gemini-viewer-examples#121]
uploading ifc files is not supported

url：
https://pattern-x.github.io/gemini-viewer-examples/#/Upload_your_local_model

console out:
[LoadingHelper] IFC is not supported directly since three.js r150!
(anonymous) @ gemini-viewer.esm.min.js:1
gemini-viewer.esm.min.js:1 Uncaught (in promise) Not supported!

Is it just me using it the wrong way?

**Comment by yanzexuan1** [comment]
You are right, ifc is not supported in this version. Three.JS removed IFCLoader from its internal package, so we need to handle ifc by ourselves. 

To do this, we can write a plugin (based on both IFC.js and gemini-viewer) that loads ifc and add the loaded object to viewer.
